### ISSUE TITLE
Improvements on YarpOpenraveControlboard

### ISSUE THREAD

**Issue by jgvictores** [roboticslab-uc3m/openrave-yarp-plugins#39]
Improvements on YarpOpenraveControlboard:
- [x] Allow loading OpenRAVE plugins.
- [x] Allow enable/disable viewer if using `env`.
- [x] Better checks on `robotIndex` and `manipulatorIndex`.
- [x] Enable `physics` parameter, something like ye oldie:
```c++
// Attach a physics engine
if(physics=="ode"){
    penv->SetPhysicsEngine(RaveCreatePhysicsEngine(penv,"ode"));
    penv->GetPhysicsEngine()->SetGravity(OpenRAVE::Vector(0,0,-9.8));
}
```

**Comment by jgvictores** [comment]
Did a small implementation of loading orPlugins at a12e8d9e13cf9725b0f57abb16ee449825307712 but only allows 1 module called exactly the same as 1 plugin.

**Comment by jgvictores** [comment]
> Allow enable/disable viewer if using env.

`view` parameter enabled at 7714d1534ba0ea2ec1748b50c24c6f444fd3ddcd

>  Allow loading OpenRAVE plugins

Two options implemented, simple and full, currently at: https://github.com/roboticslab-uc3m/openrave-yarp-plugins/blob/1e8d48466ffbd01c4db9aad085bc5077eb0d1b47/libraries/YarpPlugins/YarpOpenraveControlboard/DeviceDriverImpl.cpp#L85-L159

**Comment by jgvictores** [comment]
Added `Implement clean close function if possible` to the description.

**Comment by jgvictores** [comment]
Implemented `--physics` parameter at 82be015 (will soon be merged to `develop`).

**Comment by jgvictores** [comment]
Latest improvement merged into `develop` at 4275782b9fec1213214246640d8825ca5ef39ea9.
